### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by critesjosh**
### Description

remove baklava tag from install instructions on the ContractKit README (and npm page). add clarity to code snippets.

### Other changes

none

### CODE REVIEW

File: packages/sdk/contractkit/README.md [hunk 1]
```diff
@@ -18,19 +18,20 @@ ContractKit supports the following functionality:
...
 ```bash
-npm install @celo/contractkit@baklava
+npm install @celo/contractkit
 // or
-yarn add @celo/contractkit@baklava
+yarn add @celo/contractkit
 ```
 
-You will need node version `8.13.0` or higher.
+You will need node version `8.13.0` or higher. Node version 10 is recommended.
```
Comment (medhakothari) [inline packages/sdk/contractkit/README.md:26]: 🎉

File: packages/sdk/contractkit/README.md [hunk 2]
```diff
@@ -46,13 +47,18 @@ await kit.web3.eth.getBalance(someAddress)
...
-  const kit = newKit('https://alfajores-forno.celo-testnet.org:8545')
+async function getKit(myAddress: string, privateKey: string) {
+  const kit = newKit('https://alfajores-forno.celo-testnet.org')
 
-  // default from
+  // default from account 
   kit.defaultAccount = myAddress
+  
+  // add the account private key for tx signing when connecting to a remote node
+  kit.addAccount(privateKey)
```
Comment (medhakothari) [inline packages/sdk/contractkit/README.md]: this works, but I would change to `kit.connection.addAccount` as that is the preferred method

File: packages/sdk/contractkit/README.md [hunk 3]
```diff
@@ -63,29 +69,28 @@ celo-blockchain has two initial coins: CELO and cUSD (stableToken).
...
+const stableToken = await kit.contracts.getStableToken()
+
+const celoBalance = await goldToken.balanceOf(someAddress)
+const cusdBalance = await stableToken.balanceOf(someAddress)
 ```
 
 To send funds:
 
 ```ts
 const oneGold = kit.web3.utils.toWei('1', 'ether')
```
Comment (medhakothari) [inline packages/sdk/contractkit/README.md]: this line also still works, but similar to my comment above, would change to `kit.connection.web3.utils.toWei`

**Review by medhakothari — APPROVED** [review]
Thanks for doing this! Looks good to go except for two minor changes. They're not a blocker though, so will approve

File: packages/sdk/contractkit/README.md (hunk 2 above)
Comment (critesjosh) [reply packages/sdk/contractkit/README.md]: updated thanks

**Pull request merged**